build/CopyFilesToStagingDir.ps1

```diff
@@ -97,6 +98,7 @@ PublishFile $FullBuildOutput\WindowsAppSDK_BootstrapDLL\Microsoft.WindowsAppSDK.
 PublishFile $FullBuildOutput\WindowsAppSDK_BootstrapDLL\MddBootstrap.h $NugetDir\include
 PublishFile $FullBuildOutput\WindowsAppSDK_DLL\MsixDynamicDependency.h $NugetDir\include
 PublishFile $FullBuildOutput\WindowsAppSDK_DLL\wil_msixdynamicdependency.h $NugetDir\include
+PublishFile $FullBuildOutput\WindowsAppSDK_DLL\WindowsAppSDKInsights.h $NugetDir\include\
```

Comment:
CONSIDER: Rename to WindowsAppRuntimeInsights.h

Better fit given the naming guidance we've received (when to use SDK vs Runtime)

Our DLLs and MSIX packages are being renamed from Windows App SDK to Windows App Runtime, per guidance. Only reason they haven't been renamed yet is I held off while Will worked on some restructuring. Will be changing soon